sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -303,25 +325,42 @@ async def refresh(self, **kwargs) -> None:
                 if not self._refresh_timer.needs_refresh():
                     logging.debug("Refresh called but refresh interval not elapsed.")
                     return
+                need_refresh = False
+                updated_sentinel_keys = dict(self._refresh_on)
+                headers = _get_headers("Watch")
```

Comment:
Same comment in async version

```suggestion
                headers = _get_headers("Watch", **kwargs)
```